### PR TITLE
Improve loader transition animation at the end of add-address

### PR DESCRIPTION
**Pull request by sisou**

### CODE REVIEW

File: src/views/AddAccountSelection.vue [hunk 1]
```diff
@@ -60,10 +62,17 @@ export default class AddAccountSelection extends Vue {
 
         wallet.accounts.set(selectedAccount.userFriendlyAddress, selectedAccount);
 
-        await WalletStore.Instance.put(wallet);
-        // Artificially delay, to display loading status
-        await new Promise((res) => setTimeout(res, 1000));
+        // Display loading spinner only if storing takes longer than 500ms
+        const blockLoaderTimeout = window.setTimeout(() => this.blockLoadingSpinner = false, 500);
```
[the remark this answers is not in the document]
Comment (sisou) [reply src/views/AddAccountSelection.vue]: But this way, the button actually grows as light blue and then transitions into green success.

File: src/components/Loader.vue [hunk 2]
```diff
@@ -442,10 +442,10 @@ export default Loader;
     @keyframes loader-grow-from-bottom-button {
         0%,
         20% {
-            max-width: calc(100% - 14rem + 1rem); /* +1rem to account for button :focus effect */
+            max-width: calc(100% - 12rem);
             max-height: 7.5rem;
-            bottom: calc(4rem - var(--loader-margin) - 1px); /* -1px to account for button :focus effect */
-            left: calc(7rem - var(--loader-margin) - .5rem);
+            bottom: calc(4.25rem - var(--loader-margin)); /* -1px to account for button :focus effect */
```
[the remark this answers is not in the document]
Comment (sisou) [reply src/components/Loader.vue]: I set a fixed width style in the Page Footer component recently, which should unify the bottom buttons styles.

All it takes now should be to remove custom styling from our views.

[the remark this answers is not in the document]
Comment (sisou) [reply src/components/Loader.vue]: https://github.com/nimiq/vue-components/blob/master/src/components/PageFooter.vue#L22

**Comment by danimoh** [comment]
Missed the `showLoader` in the template when rebasing.

**Pull request merged**